src/main/frontend/src/components/landingpage/LandingPageItem.module.css

```diff
@@ -0,0 +1,98 @@
+.featureitemimg {
+  position: absolute;
+  width: 262px;
+  height: 350px;
+  display: block;
+}
+
+.full_size_picture {
+  width: 100%;
+  height: 100%;
+}
+
+.large_item_box {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  position: static;
+  width: 356px;
+  height: 543.57px;
+  left: 0px;
+  top: 0px;
+  margin-right: 24px;
+  text-align: left;
+}
```

Comment:
lots of properties inside 3 item_box classes is repeated, consider creating item_box with all common properties